static/js/public/fsf-language-select.js

```diff
@@ -50,7 +56,12 @@ function initFSFLanguageSelect(rootEl) {
         window.location.hash = "";
       }
 
-      window.scrollTo(0, link.getBoundingClientRect().top);
+      const viewportOffset = link.getBoundingClientRect().top;
```

Comment:
I'm not sure how this worked before? `getBoundingClientRect().top` is from the viewport top, no the document top? I also added `bahaviour: "smooth"` which is apparently a thing.

Reply:
Hmm... yeah, I don't know, it seems it worked in "most" cases, but I've seen it scrolling to weird places, but couldn't really reproduce it reliably.
Anyway, thanks for figuring it out and fixing.